src/TrackingLink/TrackingLink.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\TrackingModule\TrackingLink;
+
+use Exception;
+use MyParcelCom\TrackingModule\CarrierCode\CarrierCode;
+
+class TrackingLink
```

Comment:
If we have the FinalMileCarrier class we can practically remove this class because of the `FinalMileCarrier::getUrl()` method.